_posts/2020-04-20-wgpu-web.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: wgpu-rs on the web
+---
+
+[gfx-rs](https://github.com/gfx-rs/gfx) is a Rust project aiming to make low-level GPU programming portable with low overhead. It's a single Vulkan-like Rust API with multiple backends that implement it: Direct3D 12/11, Metal, Vulkan, and even OpenGL.
+
+[wgpu-rs](https://github.com/gfx-rs/wgpu-rs) is a Rust project on top of gfx-rs that provides safety, accessibility, and even stronger portability.
+
+## Running wgpu-rs natively
+
+As we previously discussed in [The rise of wgpu](http://gfx-rs.github.io/2019/03/06/wgpu.html), the gfx team has been busy with development of `wgpu-native` (a C API which implements the [WebGPU specification](https://gpuweb.github.io/gpuweb/)) and `wgpu-rs` (an idiomatic Rust wrapper around `wgpu-native`).
+
+Even though these crates are still relatively new, it's been very exciting to watch people build all kinds of neat projects which use them, including a [GUI library](https://github.com/hecrj/iced), a [pixel editor](https://github.com/cloudhead/rx), [tutorials](https://sotrh.github.io/learn-wgpu/), [multiple](https://github.com/hecrj/coffee) [game](https://github.com/Ruddle/oxidator) [engines](https://github.com/StarArawn/harmony), a [creative-coding toolkit](https://github.com/nannou-org/nannou), a [sailing navigation application](https://github.com/Yatekii/sailor), a [Brawl character file parser](https://github.com/rukai/brawllib_rs), a [hardware-accelerated pixel buffer](https://github.com/parasyte/pixels), [Python bindings](https://github.com/almarklein/wgpu-py), a [Rust vangers clone](https://github.com/kvark/vange-rs), a [Chinese calligraphy app](https://apps.apple.com/us/app/id1492608770), [Scopes bindings](https://nest.pijul.com/porky11/wgpu), and many more interesting projects. There is even an [awesome-wgpu list](https://github.com/rofrol/awesome-wgpu) that the community has started in order to aggregate wgpu-related learning resources and examples in one place.
```

Comment:
I personally prefer reading this in a bullet list.

Reply:
Good idea 👍